internal/test/wmcb/wmcb_test.go

```diff
@@ -297,6 +293,25 @@ func (vm *wmcbVM) runTestKubeletUninstall(t *testing.T) {
 	require.NoError(t, err, "TestKubeletUninstall failed")
 }
 
+// isRunning checks if the given service is running
+func (vm *wmcbVM) isRunning(serviceName string) bool {
+	svcCmd := fmt.Sprintf("sc.exe qc %s", serviceName)
```

Comment:
this is a change in functionality, checking if the process is running vs checking if the service is running and should be reflected in the commit message